library-best-practices/glossary.md

```diff
@@ -66,3 +81,55 @@ Java Dependency Glossary
 - **Linkage-matchable version** (said of a particular version of A in relation
   to all versions of B): There exists some version of B such that the version of
   A and the version of B are linkage-compatible.
+
+
+### Class usage graph
+
+- **Class usage graph**: a possibly cyclic directed graph of references between classes
+  in the classpath. The nodes (vertices) of the graph correspond to
+  Java classes and the directed edges are references between classes.
+  For example, when 'Class A' invokes method X on 'Class B',
+  the class usage graph holds an edge between the two nodes:
+
+  ```
+  [Class A] --(method X of class B)-> [Class B]
+  ```
+
+  In this case, 'Class A' is called the _source class_ of the reference and
+  'Class B' is called the _destination class_.
+
+  In general, there can be multiple (parallel) edges between two nodes when
+  a class references two or more methods and fields of another class.
+  Self-loops (references between the same class) are possible and
+  common. However, it is safe to omit such references when checking the reachability
```

Comment:
Again, here, we're leaking implementation considerations for the static linkage checker into a glossary.

Reply:
Right. Removed "However..."